compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BInvokableType.java

```diff
@@ -99,7 +105,14 @@ public String getTypeSignature() {
         if (retType.getKind() != TypeKind.NIL) {
             retTypeWithParam = "(" + retTypeWithParam + ")";
         }
-        return "(" + (paramTypes.size() != 0 ? getBTypeListAsString(paramTypes) : "") + ")"
+        String restParam = "";
+        if (restType != null && restType instanceof  BArrayType) {
```

Comment:
Formatting iassue.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/19976/commits/7432217a84fd35c7ae7682d1e9173e9416f9ccdc